services/ena.py

```diff
@@ -0,0 +1,41 @@
+from datetime import date
+from enum import Enum
+from http import HTTPStatus
+from typing import Dict, Tuple
+
+import requests
+from requests.auth import HTTPBasicAuth
+from requests.models import Response
+
+
+class EnaAction(Enum):
+    ADD = 'ADD'
+    MODIFY = 'MODIFY'
+    VALIDATE = 'VALIDATE'
```

Comment:
When would we use this action or any `VALIDATE` action?
I am not sure we would use them at all.

Reply:
This was just here to support all the options [supported by the ENA Service](https://ena-docs.readthedocs.io/en/latest/submit/general-guide/programmatic.html#submission-actions-without-submission-xml).
Technically `VALIDATE` is just an alias of `VALIDATE,ADD` but the user might have understood the ENA Docs

Reply:
If we won't need it, then please get rid of it. The simpler a code is a better code. ;-)